lib/wallet/src/Cardano/Wallet/Write/Tx/Balance/TokenBundleSize.hs

```diff
@@ -6,22 +6,21 @@ module Cardano.Wallet.Write.Tx.Balance.TokenBundleSize
     ( TokenBundleSizeAssessor (..)
     , TokenBundleMaxSize (..)
     , computeTokenBundleSerializedLengthBytes
-    , tokenBundleSizeAssessor
+    , mkTokenBundleSizeAssessor
```

Comment:
@Anviking I renamed this merely to avoid a clash with the `SelectionConstraints` field of the same name within the `Write.Tx.Balance` module. We could perhaps avoid this renaming by using a qualified import for `SelectionConstraints` instead. However, the `mk` prefix is relatively idiomatic, so perhaps it's fine?